src/services/event-service.test.ts

```diff
@@ -80,6 +81,9 @@ describe("eventService", () => {
         "2024-09-12T00:00:00.000Z"
       );
       expect(result.timestamp).to.equal(1726099200);
+      expect(result.restricted.device_information.encoded).to.equal(
+        "test-txma-header"
```

Comment:
Should we use a b64 encoded string of the length we are expecting here to make it more realistic?

Reply:
agreed, i've done this now